samples/Gerber Image/FelinaePurr-B.Cu.gbl

```diff
@@ -0,0 +1,1775 @@
+G04 #@! TF.FileFunction,Copper,L2,Bot,Signal*
```

Comment:
Could we reduce the size of these files? It looks like it's mostly the same pattern over and over, but `samples.json` is still gonna have one entry per token...